.github/workflows/ui_test.yml

```
@@ -0,0 +1,43 @@
+name: UI Test
+
+# TODO : update when to trigger this workflow as you wish
+on: push
```

```suggestion
on:
  # Run UI test every time we release
  push:
    branches:
      - main
```

I think we should specify the branch to main for release running purpose. 🙏🏻